code/obj/item/device/radios/_radio.dm

```diff
@@ -173,6 +174,15 @@ TYPEINFO(/obj/item/device/radio)
 		var/datum/say_message/global_message = message.Copy()
 		global_message.output_module_channel = SAY_CHANNEL_GLOBAL_RADIO
 		src.ensure_say_tree().process(global_message)
+		// Meta-Neural Antenna / Radiobrain
+		var/is_protected =  (src.protected_radio || !isnull(src.traitorradio) || (signal_frequency in protected_frequencies))
+		for (var/mob/radiobrain_mob as anything in radio_brains)
```

Comment:
mobs can be deleted - worth testing that the bioeffect is properly removing from this list when gibbed